src/core/classes/prism.ts

```diff
@@ -73,7 +73,6 @@ export default class Prism {
 		if (!this.config.manual) {
 			this.waitFor.push(documentReady());
 
-			this.ready = allSettled(this.waitFor);
```

Comment:
We don't need this workaround anymore: we set the initial value of `ready` on line 35.